helm_sdk/__init__.py

```diff
@@ -156,10 +157,12 @@ def uninstall(self,
                   additional_args=None,
                   **_):
 
-        if check_flag_wait_is_supported(self):
+        if self.check_flag_wait_is_supported():
             cmd = ['uninstall', name, '--wait']
         else:
             cmd = ['uninstall', name]
+            self.logger.debug('Versions of helm before 3.9.0 do not support'
```

Comment:
Let's not put this log message. I don't think it provides the user any helpful any info.